### PR TITLE
Adding change kind option

### PR DESCRIPTION
**Pull request by TheNotTooDistantFuture**

### CODE REVIEW

File: pg2kinesis/formatter.py [hunk 1]
```diff
@@ -110,24 +111,27 @@ def _preprocess_wal2json_change(self, change):
...
-                if self.full_change:
-                    changes.append(FullChange(xid=self.cur_xact, change=change))
-                else:
-                    try:
-                        full_table = '{}.{}'.format(schema, table_name)
-                        primary_key = self.primary_key_map[full_table]
-                    except KeyError:
-                        self._log_and_raise(MISSING_TABLE_ERR.format(full_table))
+
+            if change['kind'].lower() == self.change_kind.lower() or self.change_kind.lower() == 'all':
```
Comment (surbas) [inline pg2kinesis/formatter.py]: Please convert change_kind to lower case once in the `__init__`  above.

Comment (surbas) [reply pg2kinesis/formatter.py]: Is `change['kind']` ever not lower case? This is a hot loop so trying to limit unnecessary operations.

Comment (surbas) [reply pg2kinesis/formatter.py]: `all` should be check first as that is default and will short circuit the next check.

Comment (TheNotTooDistantFuture) [reply pg2kinesis/formatter.py]: Hi @surbas, thanks for the feedback! We've opened a cleaner PR with some tests included here: https://github.com/surbas/pg2kinesis/pull/21

**Pull request closed without merging**